app/plugins/adp/src/hooks/useCatalogUsersLiveSearch.tsx

```diff
@@ -64,4 +68,5 @@ export const useCatalogUsersLiveSearch = (
     },
     [allUsers.data?.items, pageSize],
   );
+  return throttle(callbackFn, DROPDOWN_THROTTLE_TIME_MS);
```

Comment:
I think throttle should be inside the callbackfn. Thats what they do in the autocomplete google maps example.
`useCallback(throttle(async (input: string) => {...}, DROPDOWN_THROTTLE_TIME_MS))`

Reply:
@danny-may I tried that initially, if we put throttle inside the callback, it creates a new throttled function on each render and hence it doesnt respect the throttle. hence have wrapped the throttle around callback to get the functionality working.